modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/InitDumpPopulation.pm

```diff
@@ -71,29 +71,17 @@ sub fetch_input {
     my $prefetched_frequencies_dir = $self->param('prefetched_frequencies');
     die "$prefetched_frequencies_dir doesn't exist" unless (-d $prefetched_frequencies_dir);
 
-    my $variation_ids_dir = "$pipeline_dir/variation_ids";
-    make_path($variation_ids_dir) unless (-d $variation_ids_dir);
-    $self->param('variation_ids_dir', $variation_ids_dir);
-
-    my $human_gvf_file_gz = $self->param('pipeline_dir') . '/gvf/Homo_sapiens/Homo_sapiens.gvf.gz';
-    die("$human_gvf_file_gz doesn't exist") unless (-f $human_gvf_file_gz);
-    my $human_gvf_file = $self->param('pipeline_dir') . '/gvf/Homo_sapiens/Homo_sapiens.gvf';
-    system("gunzip $human_gvf_file_gz");
-
-    $self->param('human_gvf_file', $human_gvf_file);
-    $self->param('gvf_files_dir', "$pipeline_dir/gvf/Homo_sapiens/");
   }
 
   if ($file_type eq 'vcf') {
-    $self->param('vcf_files_dir', "$pipeline_dir/vcf/Homo_sapiens/");
+    $self->param('vcf_files_dir', "$pipeline_dir/vcf/homo_sapiens/");
```

Comment:
Actually wondering if `$pipeline_dir/vcf/homo_sapiens/` should be setup/declared as a pipeline variable